boot/initramfs20dirs.go

```diff
@@ -50,18 +50,19 @@ var (
 	// initramfs.
 	InitramfsWritableDir string
 
-	// InitramfsWritableDir is the location of the encrypted partition keys
+	// InitramfsEncryptionKeyDir is the location of the encrypted partition keys
 	// during the initramfs.
 	InitramfsEncryptionKeyDir string
 )
 
 func setInitramfsDirVars(rootdir string) {
 	InitramfsRunMntDir = filepath.Join(rootdir, "run/mnt")
-	InitramfsUbuntuDataDir = filepath.Join(InitramfsRunMntDir, "ubuntu-data")
+	InitramfsDataDir = filepath.Join(InitramfsRunMntDir, "data")
+	// TODO:UC20 remove the compatibility mount once the transition period is over
 	InitramfsHostUbuntuDataDir = filepath.Join(InitramfsRunMntDir, "host", "ubuntu-data")
 	InitramfsUbuntuBootDir = filepath.Join(InitramfsRunMntDir, "ubuntu-boot")
 	InitramfsUbuntuSeedDir = filepath.Join(InitramfsRunMntDir, "ubuntu-seed")
-	InitramfsWritableDir = filepath.Join(InitramfsUbuntuDataDir, "system-data")
+	InitramfsWritableDir = filepath.Join(InitramfsDataDir, "system-data")
```

Comment:
is this change correct? won't create partitions code mounting use /ubuntu-data ?

Reply:
as discussed over IRC, this change was slightly incorrect and we modified it to have 2 vars now